### DISCUSSION TITLE
training video

### DISCUSSION THREAD

**Discussion by cherfffff** [deforum-art/sd-webui-text2video#194] [Q&A]
I want to train my own video model, please
1. How long should I cut each video into? How many frames per video? How many videos are needed?
2. After I have finished training, how to call the model in the webui?

**Comment by B34STW4RS** [comment]
So far the only good custom model was trained with this amount of data:

zeroscope_v2_576

9,923 clips and 29,769 tagged frames at 24 frames, 576x320 resolution
